content/en/docs/Advanced/revoking-secureboot-access.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Revoking secure boot access"
+linkTitle: "Revoking secure boot access"
+weight: 10
+date: 2022-06-12
+---
+
+This document describes how an administrator can prevent certain OS images from
+booting on their hardware in the context of ["Trusted Boot"]({{< relref "../architecture/trustedboot" >}}).
+
+Two different scenarios will be covered, with the process being only slightly different for each case.
+
+## Scenario 1 - Signing certificate no longer trusted
```

Comment:
what does this actually mean, that you sign a certificate which is no longer trusted? or that a signed certificate is no longer trusted?

Reply:
The latter. Maybe ` Signing certificate is no longer trusted` ?

Reply:
yup sounds good to me